webui_runner/Dockerfile

```diff
@@ -1,31 +1,31 @@
 FROM ubuntu:bionic
 
 RUN apt-get update \
- && apt-get install -y --no-install-recommends \
+	&& apt-get install -y --no-install-recommends \
 	apt-utils build-essential gpg-agent \
 	curl ca-certificates wget software-properties-common \
-    psmisc lsof git cmake nano
+	psmisc lsof git cmake nano
 
 RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key|apt-key add -
 
-RUN add-apt-repository "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-8 main"
+RUN add-apt-repository "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-10 main"
 
 RUN apt-get update \
 	&& apt-get install -y --no-install-recommends \
-	clang-8 lldb-8 lld-8
-
-RUN curl -sL https://github.com/CraneStation/wasi-sdk/releases/download/wasi-sdk-6/libclang_rt.builtins-wasm32-wasi-6.0.tar.gz  | tar x -zf - -C /usr/lib/llvm-8/lib/clang/8.0.1
+	clang-10 lldb-10 lld-10
 
-RUN ln -s /usr/bin/wasm-ld-8 /usr/bin/wasm-ld
+RUN curl -k -O -L https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-11/wasi-sdk_11.0_amd64.deb
+RUN apt install ./wasi-sdk_11.0_amd64.deb
 
 ENV LD_LIBRARY_PATH=/usr/local/lib
 
-RUN curl -sS -L -O https://static.rust-lang.org/dist/rust-1.37.0-x86_64-unknown-linux-gnu.tar.gz \
-	&& tar xzf rust-1.37.0-x86_64-unknown-linux-gnu.tar.gz \
-	&& cd rust-1.37.0-x86_64-unknown-linux-gnu \
+# https://forge.rust-lang.org/infra/other-installation-methods.html
+RUN curl -sS -L -O https://static.rust-lang.org/dist/rust-1.44.1-x86_64-unknown-linux-gnu.tar.gz \
+	&& tar xzf rust-1.44.1-x86_64-unknown-linux-gnu.tar.gz \
+	&& cd rust-1.44.1-x86_64-unknown-linux-gnu \
 	&& ./install.sh \
 	&& cd .. \
-	&& rm -rf rust-1.37.0-x86_64-unknown-linux-gnu rust-1.37.0-x86_64-unknown-linux-gnu.tar.gz
+	&& rm -rf rust-1.44.1-x86_64-unknown-linux-gnu rust-1.44.1-x86_64-unknown-linux-gnu.tar.gz
```

Comment:
What do you think about adding a `RUST_VERSION` variable at the top of this file and using it throughout to make the next upgrade easier? Same idea probably applies to the other installed components.